packages/frontend-2/utils/dateFormatter.ts

```diff
@@ -65,7 +65,7 @@ export const formattedRelativeDate = (
   if (options?.prefix) {
     return isClockUnit(date)
       ? customRelativeTime(date, options?.capitalize)
-      : `on ${customRelativeTime(date)}`
+      : `${customRelativeTime(date)}`
```

Comment:
Why? What's the point of the `prefix` option then, if it doesn't add a prefix? If we no longer want the prefix, then we should remove the option instead of having a confusing option called `prefix` that doesn't actually add a prefix.

Reply:
Same here, we should have the `on` still in there

Reply:
There is a bug here, but I have reverted my change, and @Mikehrn will take a look at the bug in it's own branch